### ISSUE TITLE
Missing Campaign Progress Widget for Donate (XenForo 2)

### ISSUE THREAD

**Issue by jljr222** [Audentio/xf2addon-issues#160]
Currently missing the ability to natively add campaign progress to the widgets for appearance. I believe a widget for Campaign Progress and Recent Donors would be awesome. I would priority the campaign progress. Thanks!

**Comment by mikecreuzer** [comment]
Im pretty sure we did build this, it'd be a huge oversight otherwise, so I wonder if for some reason it didn't get associated when we exported the add-on @JakeBooher

**Comment by JakeBooher** [comment]
Yeah, looks like they were incorrect associated. Will be resolved in 1.0.1!
